compair/api/report.py

```diff
@@ -488,7 +573,10 @@ def peer_feedback_report(course, assignments, group):
                     else:
                         temp += [feedback.receiver_lastname, feedback.receiver_firstname, feedback.receiver_student_number]
 
-                    temp += [feedback_type, strip_html(feedback.content)]
+                    plain_feedback_content = strip_html(feedback.content).strip()
```

Comment:
maybe use `.strip(' \t\n\r')` to get rid of leading/trailing newlines tabs and spaces